WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -169,6 +169,20 @@ class PostCoordinator: NSObject {
         }
     }
 
+    /// Patches the post but keeps the local revision if any.
+    ///
+    /// - warning: Work-in-progress (kahu-offline-mode)
+    @MainActor
+    func _update(_ post: AbstractPost, changes: RemotePostUpdateParameters) async throws {
```

Comment:
This is the simplest possible form of updating the post: send diff, update the `original` post with `RemotePost` from the server. Keeps the local revision intact. It'll be required in a few places when working with published posts.